requirements/common.txt

```diff
@@ -336,8 +336,8 @@ mo-json==3.67.20113 \
 mo-kwargs==3.54.20059 \
     --hash=sha256:f6ada2f6e84a700db6cbdc5901c12fcba351005440c8aa3f256dbf17fd3eef88 \
     # via jx-bigquery, jx-mysql, jx-python, mo-collections, mo-http
-mo-logs==3.66.20113 \
-    --hash=sha256:49d99312f35e84db47e48897b3d61e4abc1c6419868025eab827c3f57ff53480 \
+mo-logs==3.75.20189 \
+    --hash=sha256:d0cc393653b0ba11b4506960258236030baee20b8706253605c83cd82bb9067f \
```

Comment:
This is a required upgrade - so that the exception that extract_alerts throws is flushed to stdout (and picked up by papertrail)